java/src/main/java/com/google/oak/crypto/ClientEncryptor.java

```diff
@@ -59,8 +64,22 @@ public class ClientEncryptor implements Encryptor {
    * <https://secg.org/sec1-v2.pdf>
    */
   // TODO(#3642): Implement Java Hybrid Encryption.
-  public ClientEncryptor(final byte[] serializedServerPublicKey) {
-    this.serializedServerPublicKey = serializedServerPublicKey;
+  public static final Result<ClientEncryptor, Exception> create(
+      final byte[] serializedServerPublicKey) {
```

Comment:
I think this whole method can be simplified to 

```
    return Hpke.setupBaseSender(serializedServerPublicKey, OAK_HPKE_INFO).map(ClientEncryptor::new);
```

Reply:
Done